.bazelrc

```diff
@@ -40,11 +40,10 @@ build --use_target_config_carbon_rules
 # Default to using a disk cache to minimize re-building LLVM and Clang which we
 # try to avoid updating too frequently to minimize rebuild cost. The location
 # here can be overridden in the user configuration where needed.
-#
-# Note that this cache will grow without bound currently. You should
-# periodically run the `scripts/clean_disk_cache.sh` script or some equivalent.
-# https://github.com/bazelbuild/bazel/issues/5139 tracks fixing this in Bazel.
 build --disk_cache=~/.cache/carbon-lang-build-cache
+# If you'd like a different disk cache size, override it by copying this
+# line to `user.bazelrc` in the repository root and modify the number there.
+build --experimental_disk_cache_gc_max_size 100G
```

Comment:
```suggestion
build --experimental_disk_cache_gc_max_size=100G
```

Reply:
BTW, should this (and `--disk_cache`) also be using `common` instead of `build`? Maybe try switching both, particularly if things like `bazel query //...` and `bazel info` work with this as `common`